### PR TITLE
Barebones implementation of AddInsuranceCommand

### PR DESCRIPTION
**Pull request by yanqiyqh**
Add AddInsuranceCommand and hook AddInsuranceCommand into the application
(no exception, no enhancement)

### CODE REVIEW

**Comment by yanqiyqh** [comment]
for issue #43

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java [hunk 1]
```diff
@@ -19,6 +13,13 @@
 import seedu.address.logic.commands.ListCommand;
 import seedu.address.logic.parser.exceptions.ParseException;
 
+import java.util.logging.Logger;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.Messages.MESSAGE_UNKNOWN_COMMAND;
+
```
Comment (RezwanAhmed123) [inline src/main/java/seedu/address/logic/parser/AddressBookParser.java:22]: should follow this following [order](https://se-education.org/guides/tutorials/intellijCodeStyle.html#tweak-import-order), so static imports should have been on top

**Review by RezwanAhmed123 — APPROVED** [review]
LGTM, I'll fix the import orders later when i continue on the work

**Comment by RezwanAhmed123** [comment]
I'll merge as the changes are minimal and the only failing tests are the ones in checkstyle.

**Pull request merged**